aviary/utils/doctape.py

```diff
@@ -89,6 +89,7 @@ def get_previous_line(n=1) -> str:
 def get_variable_name(*variables) -> str:
     """
     returns the name of the variable passed to the function as a string
+    # NOTE: You cannot call this function multiple times on one line
```

Comment:
In python 3.11 [inspect](https://docs.python.org/3.11/library/inspect.html#inspect.FrameInfo) adds `positions` as an attribute to `FrameInfo` that will allow for this